<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1196</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
                           margin-top: 6px;
}
QGroupBox::title {
                            subcontrol-origin: margin;
                            subcontrol-position: top left;
                            margin-top: 0px;
                            margin-left: 10px;
                            background-color: palette(window);
                            }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelObjectHierarchyName">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidgetObjectHierarchy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="lineEditObjectHierarchyName">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxObject">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>551</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

</string>
    </property>
    <property name="title">
     <string>Object</string>
    </property>
    <widget class="QLabel" name="labelObjectName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Object name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditObjecName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>221</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>511</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditObjecUUID">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>221</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelObjectUUID">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>UUID:</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableViewObject">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>31</y>
      <width>331</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1196</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
